dbt/macros/res_report_summary.sql

```diff
@@ -0,0 +1,53 @@
+-- Macros that summarize FMVs and sales by different groupings specifically for
+-- reporting.res_report_summary.
+{% macro res_report_summarize_values(vals, geo_type, prop_group) %}
```

Comment:
**[Nitpick, non-blocking]** I find `vals` and `sales` to be a bit confusing as parameter names, and I don't think I would have immediately known what they were for without reading the source code of the `res_report_summarize_values` macro. A clearer parameter name might be something like `from`, to indicate that the argument will be used as the source table for the summary query, but this is more of a personal preference so I don't feel strongly about it.

Reply:
I like it.